DigitalLearningSolutions.Web/Views/SuperAdmin/Faqs/_FaqCard.cshtml

```diff
@@ -0,0 +1,60 @@
+<div class="searchable-element nhsuk-expander-group word-break" id="@Model.FaqId-card">
+
+  <details class="nhsuk-details nhsuk-expander">
+    <div class="nhsuk-details__text">
+      <div class="nhsuk-grid-row">
+        <div class="nhsuk-grid-column-one-quarter">
+          <div class="tag">
+            @Model.TargetGroup
+          </div>
+        </div>
+        <div class="nhsuk-grid-column-one-quarter">
+          <div class="tag">
+            @Model.Published
+          </div>
+        </div>
+      </div>
+    </div>
+    <summary class="nhsuk-details__summary">
+      <span class="nhsuk-details__summary-text" id="@Model.FaqId-name">
+        <span class="searchable-element-title" name="name">@Model.QText</span>
+      </span>
+    </summary>
+
+    <div class="nhsuk-details__text">
+      <div class="nhsuk-u-margin-bottom-4">
+        <span class="searchable-element-content">@Html.Raw(Model.AHtml)</span>
+
+        <div class="nhsuk-grid-row">
+          <div class="nhsuk-grid-column-one-quarter">
+            <div class="nhsuk-u-font-weight-bold">
+              <p>Created</p>
+              <p>Weighting</p>
+            </div>
+          </div>
+          <div class="nhsuk-grid-column-one-quarter">
+            <p>@Model.CreatedDate</p>
+            <p>@Model.Weighting</p>
+          </div>
+        </div>
+      </div>
+
+      <div>
+        <a class="nhsuk-button nhsuk-button--danger"
```

Comment:
This I think should be on the same line as the others. The wireframe is out of date but it is mentioned in the acceptance criteria.